test/functional/gridfs_stream_tests.js

```diff
@@ -1188,15 +1188,20 @@ describe('GridFS Stream', function() {
 
                 download.on('end', function() {
                   var result = testSpec.assert.result;
-                  expect(result).to.exist;
-                  test.equal(res.toString('hex'), result.$hex);
+                  if (result) {
```

Comment:
Why did this change to using an if statement? `expect(result).to.exist` should cover this.

Reply:
We wanted to make sure that it would run `download.abort()` and `client.close()` if the result does not exist

Reply:
We skipped the incorrect tests and removed the if statement.